.vscode/tasks.json

```diff
@@ -31,10 +31,18 @@
                     "-c",
                     "Debug",
                     "FSharp.Compiler.Service.sln"
-                ],
+                ]
+            },
+            "options": {
+                "env": {
+                    "UpdateXlfOnBuild": "true"
+                }
             },
             "problemMatcher": "$msCompile",
-            "group": "build",
+            "group": {
+                "kind": "build",
+                "isDefault": true
+            }
```

Comment:
Same here, meant to do that for a long time. But didn't have a PR for that. Default build just makes sense to have, and update xlfs on rebuild just makes sense.